tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -175,6 +175,13 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
                 }
             }
 
+            process.EnvironmentVariables.TryGetValue("DD_TRACE_ENABLED", out var traceEnabledValue);
+
+            if (traceEnabledValue == "false")
```

Comment:
Agreed, you can use the `ParseBooleanConfigurationValue()` helper I think:

```suggestion
            if (process.EnvironmentVariables.TryGetValue("DD_TRACE_ENABLED", out var traceEnabledValue) 
              && !ParseBooleanConfigurationValue(traceEnabledValue))
```